src/lib/cat_superhero.svelte

```diff
@@ -381,23 +497,84 @@
     }
     /* Roadmap, markers, and text */
     .roadmap {
-      background-image: url(@base/roadmap/road.svg);
+      display: block;
+      background-image: none;
       background-repeat: no-repeat;
       background-size: auto 100%;
-      padding-top: 6rem;
+      padding-top: 18rem;
       padding-bottom: 5vw;
       margin: 0 1rem;
+      position: relative;
+    }
+
+    .toggle-button {
+      width: 0;
+      height: 0;
+      border-left: 10px solid transparent;
+      border-right: 10px solid transparent;
+      border-top: 10px solid var(--color-secondary-accent);
+      transition: transform 0.2s ease-in;
     }
+
+    .toggle-button[aria-expanded='true'] {
+      transform: scaleY(-1);
+    }
+
+    .toggle-button:hover {
+      cursor: pointer;
+    }
+
+    .accordion-image {
+      width: 100%;
+    }
+
+    .external-link-icon {
+      width: 14px !important;
+      height: 14px !important;
+      margin-bottom: 3px;
+    }
+
+    .link-no-underline {
+      text-decoration: none;
+    }
+
+    .road-image {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      position: absolute;
+      top: 6rem;
+      left: 0;
+      border-radius: 90px;
+      background: var(--color-primary);
+      height: calc(100% - 6rem);
+      width: 60px;
+    }
+
+    .road-line {
+      height: 100%;
+      width: 4px;
+      background-image: url(@base/roadmap/road_border.png);
```

Comment:
I'm assuming the previous solution (with everything embedded in an SVG) wasn't working -- we can keep this if you've played around and concluded it's best, but on my Mac's retina display this image gets stretched horizontally until it's blurry (since it's only 5px).

(One thought is just to use `border-left: 5px dotted white` on this element, instead of setting its `background-image`)